skule_vote/backend/views.py

```diff
@@ -149,8 +149,10 @@ def get(self, request, *args, **kwargs):
         except IncompleteVoterInfoError:
             return HttpResponse(status=400)
 
-        # TODO: redirect to frontend
-        res = HttpResponseRedirect(reverse_lazy("api:backend:election-list"))
+        if settings.DEBUG or not settings.CONNECT_TO_UOFT:
+            res = HttpResponseRedirect(reverse_lazy("api:backend:election-list"))
+        else:
+            res = HttpResponseRedirect("/elections")
```

Comment:
This is not going to work because it redirects to `vote.skule.ca/elections` through the Apache proxy, however `/elections` is an endpoint *within* the React app, but the Apache proxy doesn't know that. You have to find a way to tell the React app what you want.


You can see the Apache error when you try going to `vote.skule.ca/elections` right now before this is deployed.

Reply:
Appears to be resolved using nginx.

Changed `try_files $uri $uri/ =404;` to `try_files $uri /index.html;`